components/brave_wallet/browser/zcash/rust/lib.rs

```diff
@@ -2,14 +2,32 @@
 // This Source Code Form is subject to the terms of the Mozilla Public
 // License, v. 2.0. If a copy of the MPL was not distributed with this file,
 // You can obtain one at https://mozilla.org/MPL/2.0/.
+
 use std::fmt;
 
 use orchard::{
+    builder:: {
+        BuildError as OrchardBuildError,
+        InProgress,
+        Unproven,
+        Unauthorized
+    },
+    bundle::Bundle,
     zip32::ChildIndex as OrchardChildIndex,
     keys::Scope as OrchardScope,
     keys::FullViewingKey as OrchardFVK,
     zip32::Error as Zip32Error,
-    zip32::ExtendedSpendingKey};
+    zip32::ExtendedSpendingKey,
+    tree::MerkleHashOrchard
+};
+
+use zcash_primitives::transaction::components::amount::Amount;
+
+use ffi::OrchardOutput;
+
+use rand_core::OsRng;
```

Comment:
the orchard crate opts to use `rand` instead of `rand_core`. I see this OsRng works for this, but it's not clear what the reasoning for switching is. Could you explain the rationale for switching so we have it documented in this PR review?

Reply:
Switched to use rand instead of rand_core, just reused gen function from rand_core::impls

Reply:
This LGTM